webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1955,7 +1955,7 @@ var userOverride = {
             var $mck_tab_individual = $applozic('#mck-tab-individual');
             var MCK_IDLE_TIME_COUNTER = MCK_IDLE_TIME_LIMIT;
             var INITIALIZE_APP_URL = '/v2/tab/initialize.page';
-            var FEEDBACK_UPDATE_URL = '/feedback/v2';
+            var FEEDBACK_UPDATE_URL = '/rest/ws/feedback/v2/v2';
```

Comment:
is v2 extra here?

Reply:
yes, in the backend new secure route was `/rest/ws/feedback/v2`
and this API earlier was `/feedback/v2`
so when we replaced `/feedback` with `/rest/ws/feedback/v2`, it got mixed up. and 2 `v2` got added.
once all the old routes are refactored from all SDKs, we can correct this.

Reply:
or add support for v3 and use the same in this PR and add support for both v3 and /v2 and /v2/v2

Reply:
means in backend add support for `/v3/v2` and use that here?

Reply:
no i mean, we need to avoid v2/v2, and add support for /v3 as well [just v3 and only once, no duplicate things]

and keep things simple and understandable

Reply:
okay. but for that, we need to do backend changes as well. is that okay? i mean, it will then create a dependency of deployment order.

Reply:
ok then leave it as same